waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -246,8 +246,9 @@
 
 (defn default-marathon-descriptor-builder
   "Returns the descriptor to be used by Marathon to create new apps."
-  [home-path-prefix service-id->password-fn {:keys [service-id service-description]} _]
-  (let [health-check-url (sd/service-description->health-check-url service-description)
+  [home-path-prefix service-id->password-fn {:keys [service-id service-description]} {:keys [container-init-commands]}]
+  (let [base-command-vector (vec (or container-init-commands ["waiter-mesos-init"]))
```

Comment:
Should we allow empty/missing `container-init-commands` instead of providing a default here? Instead, we can require that `container-init-commands` ne non-empty instead of providing a default.

Reply:
Done!